src/main/java/org/Yamini_1.java

```diff
@@ -0,0 +1,6 @@
+package org;
+
+public class Yamini_1 {
+
```

Comment:
super - add comment

Reply:
This is for training , will add big pgms soon